spec/facility_spec.rb

```diff
@@ -2,25 +2,90 @@
+    @bolt = Vehicle.new({ vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev })
+    @camaro = Vehicle.new({ vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice })
+    @registrant_1 = Registrant.new({name: "Bruce", age: 18, permit: true})
+    @registrant_2 = Registrant.new({name: "Penny", age: 15, permit: false})
+    @registrant_3 = Registrant.new({name: "Tucker", age: 15, permit: false})
   end
+
   describe '#initialize' do
     it 'can initialize' do
-      expect(@facility).to be_an_instance_of(Facility)
-      expect(@facility.name).to eq('DMV Tremont Branch')
-      expect(@facility.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
-      expect(@facility.phone).to eq('(720) 865-4600')
-      expect(@facility.services).to eq([])
+      expect(@facility_1).to be_an_instance_of(Facility)
+      expect(@facility_1.name).to eq('DMV Tremont Branch')
+      expect(@facility_1.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
+      expect(@facility_1.phone).to eq('(720) 865-4600')
+      expect(@facility_1.services).to eq([])
+      expect(@facility_2).to be_an_instance_of(Facility)
+      expect(@facility_2.name).to eq('DMV Northeast Branch')
+      expect(@facility_2.address).to eq('4685 Peoria Street Suite 101 Denver CO 80239')
+      expect(@facility_2.phone).to eq('(720) 865-4600')
+      expect(@facility_2.services).to eq([])
     end
   end
 
   describe '#add service' do
     it 'can add available services' do
-      expect(@facility.services).to eq([])
-      @facility.add_service('New Drivers License')
-      @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+      expect(@facility_1.services).to eq([])
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.services).to eq(['Vehicle Registration'])
+      @facility_1.add_service('Written Test')
+      expect(@facility_1.services).to eq(['Vehicle Registration', 'Written Test'])
+
+      
+      expect(@facility_2.services).to eq([])
+    end
+  end
+
+  describe '#register_a_vehicle' do
+    it 'can add a registered vehicle to list of registered vehicles' do
+      @facility_1.add_service("Vehicle Registration")
+      expect(@facility_1.registered_vehicles).to eq([])
+      @facility_1.register_a_vehicle(@cruz)
+      expect(@facility_1.registered_vehicles).to eq([@cruz])
+      
+      @facility_1.register_a_vehicle(@camaro)
+      expect(@facility_1.registered_vehicles).to eq([@cruz, @camaro])
+      
+      @facility_1.register_a_vehicle(@bolt)
+      expect(@facility_1.registered_vehicles).to eq([@cruz, @camaro, @bolt])
+
+      expect(@facility_2.registered_vehicles).to eq([])
+      @facility_2.register_a_vehicle(@bolt)
+      expect(@facility_2.registered_vehicles).to eq([])
```

Comment:
This is a good start to this test, to make sure we can not register a vehicle if the facility does not offer this service. Then, after line 60, you'd want to repeat line 57 and then make sure that you DO see @bolt when you call @facility.registered_vehicles.